packages/react/src/themes/teams/components/Menu/menuStyles.ts

```diff
@@ -38,14 +42,14 @@ export default {
         !iconOnly &&
         !(pointing && vertical) &&
         !underlined && {
-          border: `${v.borderWidth} solid ${v.borderColor}`,
-          ...(primary && {
-            border: `${v.borderWidth} solid ${v.primaryBorderColor}`,
-          }),
+          // primary has hardcoded grey border color
+          border: `${v.borderWidth} solid ${
+            primary ? v.primaryBorderColor : v.borderColor || colorScheme.border
```

Comment:
This is an exercise on operator precedence

Reply:
My intention was this: `primary ? v.primaryBorderColor : (v.borderColor || colorScheme.border)`  Is this more obvious? :)